### PR TITLE
Mention contributor playground in the contributor guide

### LINKED ISSUE #3372: Contributor guide needs to mention the playground
The contributor guide should mention the playground so people can "practice" doing pull requests. 

https://github.com/kubernetes-sigs/contributor-playground/

/sig contributor-experience
/area contributor-guide
/help
/good-first-issue

### PR DESCRIPTION
**Pull request by nzoueidi**
Fixes #3372 and #2100

### CODE REVIEW

**Comment by nzoueidi** [comment]
cc @castrojo @parispittman

**Comment by castrojo** [comment]
/lgtm

This looks great to me, thanks for your contribution!

File: contributors/guide/README.md [hunk 1]
```diff
@@ -251,6 +253,14 @@ Have you ever noticed the total number of [open issues](https://issues.k8s.io)?
 Helping to manage or triage these open issues can be a great contribution and a great opportunity to learn about the various areas of the project. Triaging is the word we use to describe the process of adding multiple types of descriptive labels to GitHub issues, in order to speed up routing issues to the right folks.
 Refer to the [Issue Triage Guidelines](/contributors/guide/issue-triage.md) for more information.
 
+# Kubernetes Contributor Playground
+
+If you are looking for a safe place, where you can familiarize yourself with (some of) the Kubernetes Project's review and pull request processes. Then the [Kubernetes Contributor Playground](https://github.com/kubernetes-sigs/contributor-playground/) is the right place for you.
```
Comment (nikhita) [inline contributors/guide/README.md]: nit: maybe `Then..` doesn't need to be a new sentence?

File: contributors/guide/README.md [hunk 2]
```diff
@@ -251,6 +253,14 @@ Have you ever noticed the total number of [open issues](https://issues.k8s.io)?
...
+
+## Youtube playlist
+
+A [Youtube playlist](https://www.youtube.com/playlist?list=PL69nYSiGNLP3M5X7stuD7N4r3uP2PZQUx) of the New Contributor workshop has been posted, and an outline of content to videos can be found [here](https://github.com/kubernetes/community/tree/master/events/2018/05-contributor-summit).
```
Comment (nikhita) [inline contributors/guide/README.md]: for the link, please use `/events/2018/05-contributor-summit` for consistency :)

Comment (nzoueidi) [reply contributors/guide/README.md]: Thanks for this, I updated it.

File: contributors/guide/README.md (hunk 1 above)
Comment (nzoueidi) [reply contributors/guide/README.md]: you are right, just updated it.

File: contributors/guide/README.md [hunk 3]
```diff
@@ -251,6 +253,14 @@ Have you ever noticed the total number of [open issues](https://issues.k8s.io)?
 Helping to manage or triage these open issues can be a great contribution and a great opportunity to learn about the various areas of the project. Triaging is the word we use to describe the process of adding multiple types of descriptive labels to GitHub issues, in order to speed up routing issues to the right folks.
 Refer to the [Issue Triage Guidelines](/contributors/guide/issue-triage.md) for more information.
 
+# Kubernetes Contributor Playground
+
+If you are looking for a safe place, where you can familiarize yourself with (some of) the Kubernetes Project's review and pull request processes, then the [Kubernetes Contributor Playground](https://github.com/kubernetes-sigs/contributor-playground/) is the right place for you.
+
+## Youtube playlist
+
+A [Youtube playlist](https://www.youtube.com/playlist?list=PL69nYSiGNLP3M5X7stuD7N4r3uP2PZQUx) of the New Contributor workshop has been posted, and an outline of content to videos can be found [here](/events/2018/05-contributor-summit).
```
Comment (nikhita) [inline contributors/guide/README.md]: sorry for not catching this earlier...tiny nit: s/outline of content to videos/outline of the video content can be found...

**Comment by nikhita** [comment]
/assign @castrojo 
for approval

**Pull request merged**